payara-maven/README.md

```diff
@@ -0,0 +1,86 @@
+# Demonstration of using Payara Server with the Maven Cargo plugin
+
+## Quick start
+
+Start Payara server at the standard admin port 4848 and HTTP listener port 8080 and run:
+
+```
+mvn clean install
+```
+
+This profile `payara5x-remote` will be enabled by default.
```

Comment:
Did you mean "The profile..." ? "This profile..." sounds like it was mentioned before but it wasn't, it's a bit confusing.